src/main/drivers/vtx_rtc6705.c

```diff
@@ -56,10 +56,10 @@
 static IO_t vtxPowerPin     = IO_NONE;
 #endif
 
-static busDevice_t *busdev = NULL;
+static extDevice_t *dev = NULL;
```

Comment:
Has this driver been tested? (N.b. the RTC6705 does not properly implement the SPI protocol.)

Reply:
No, but I believe it'll be fine as it's write only and as a MOSI only device looks like it will act normally. Are you able to test, or do I need to ask around?

Reply:
As noted in the comments, this is broken as the rtc6705 softspi driver doesn't use the SPI API.